answer/ponpain/CSS/01/centerDiv.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>centerDiv</title>
+    <style>
+        div{
```

Comment:
```suggestion
        div {
```
セレクタとブラケットはスペースを開けましょう